### PR TITLE
Create the orbit-simrel repository

### PR DESCRIPTION
**Pull request by merks**
https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3198

### CODE REVIEW

File: otterdog/eclipse-orbit.jsonnet [hunk 1]
```diff
@@ -51,5 +51,14 @@ orgs.newOrg('eclipse-orbit') {
       secret_scanning_push_protection: "disabled",
       web_commit_signoff_required: false,
     },
+    orgs.newRepo('orbit-simrel') {
+      allow_merge_commit: false,
+      allow_update_branch: false,
+      description: "Third-party libraries bundled using Maven target locations.",
+      has_wiki: false,
+      secret_scanning: "disabled",
```
Comment (netomi) [inline otterdog/eclipse-orbit.jsonnet]: I would suggest to enable secret scanning and secret scanning push protection, especially for new repos, if you remove these lines, the values from the default config are used, which enable it.

The same applies for the already existing repos. The config was created with the status quo, but we strongly suggest to enable these settings.

Comment (merks) [reply otterdog/eclipse-orbit.jsonnet]: Good suggestion. 👍   Done.

**Comment by merks** [comment]
This is really cool!  Thanks.

**Comment by netomi** [comment]
Changes have been applied. It might take 1-2 hours for the permissions setup for the new repo as the sync script runs approx. every 2 hours.

**Comment by merks** [comment]
Good to know. I’m in no rush.

**Pull request merged**